libs/@wingcloud/framework/src/main.ts

```diff
@@ -1,5 +1,7 @@
 import { platform, std } from "@winglang/sdk";
-import { Construct } from "constructs";
+import type {
+  Construct
+} from "@winglang/sdk/lib/core/types";
```

Comment:
Does a user also need to do this if they're creating Construct classes their app code? (wondering if the `empty` template needs to be updated as well)

Reply:
Hmm good point. I need to do some experiments to see what causes the least friction.